test/MINLPTests/Project.toml

```diff
@@ -12,6 +13,6 @@ Bonmin_jll = "100.800.801"
 Couenne_jll = "0.500.801"
 Ipopt_jll = "=300.1400.400"
 JuMP = "1"
-MINLPTests = "0.5"
+MINLPTests = "0.5, 0.6"
```

Comment:
When new version of MINLPTests is released, we should remove the `0.5` and get rid of the `Pkg` stuff